src/pages/index.js

```diff
@@ -21,23 +21,23 @@ export default function Home() {
 					siteName: NEXT_APP_NAME,
 				}}
 			/>
-			<div className="flex align-middle justify-center">
-				<div className="py-16 text-center">
+			<div className="flex flex-col sm:flex-row justify-center items-center">
```

Comment:
And how about `sm:h-screen`?

Reply:
Will try tonight. There is few changes to add ser!

Reply:
> And how about sm:h-screen?

Problem with h-screen, is that it will add 100vh on the div from index.js. This is not good.
We should add #40 :)

Reply:
Yes... but overall it's nice and well done!! :fire: